docs/content/labs/common/tls-certificates.md

```diff
@@ -0,0 +1,45 @@
+!!! important
+
+    This section is only relevant in disconnected scenarios. If this doesn't apply to your situation, please proceed to the next section.
+
+In this section, we'll cover the TLS certificates involved in the process, primarily focusing on the private registries from which the images will be pulled. While there may be additional certificates, we'll concentrate on these particular ones.
+
+It's important to distinguish between the various methods and their impact on the associated cluster. All of these methods essentially modify the content of the following files on the OCP (OpenShift Container Platform) master and worker nodes:
```

Comment:
@jparrill If I recall correctly, I believe the terminology changed from "master" to "control plane" in OCP.